locales/en.js

```
@@ -205,7 +205,40 @@ const en = {
     "restorecompleted": "Restore completed successfully!",
     "restorefailed": "Restore failed: ",
     "invalidBackup": "Invalid backup file selected.",
+    "webdavTitle": "WebDAV Sync",
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)